GVFS/GVFS.Common/Maintenance/GitMaintenanceScheduler.cs

```diff
@@ -9,6 +9,10 @@ public class GitMaintenanceScheduler : IDisposable
     {
         private readonly TimeSpan looseObjectsDueTime = TimeSpan.FromMinutes(5);
         private readonly TimeSpan looseObjectsPeriod = TimeSpan.FromHours(6);
+
+        private readonly TimeSpan packfileDueTime = TimeSpan.FromMinutes(10);
+        private readonly TimeSpan packfilePeriod = TimeSpan.FromHours(12);

```

Comment:
Should we pick something that will not collide with the 6 hour timer for looseObjects?

Reply:
they will be 5 minutes off due to the `dueTime` difference, but also it's fine if they both get added to the queue (they won't be running at the same time).